src/core/lib/iomgr/ev_epollex_linux.cc

```diff
@@ -550,6 +567,8 @@ static grpc_error* pollable_create(pollable_type type, pollable** p) {
   gpr_mu_init(&(*p)->mu);
   (*p)->epfd = epfd;
   (*p)->owner_fd = nullptr;
+  gpr_mu_init(&(*p)->owner_orphan_mu);
```

Comment:
We should have a corresponding ```gpr_mu_destroy``` somewhere.

Reply:
Done.